src/main/java/de/mediathekview/mserver/crawler/sr/tasks/SrTopicArchivePageTask.java

```diff
@@ -0,0 +1,67 @@
+package de.mediathekview.mserver.crawler.sr.tasks;
+
+import de.mediathekview.mserver.base.Consts;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractUrlTask;
+import de.mediathekview.mserver.crawler.sr.SrTopicUrlDTO;
+import de.mediathekview.mserver.crawler.sr.SrConstants;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+public class SrTopicArchivePageTask extends AbstractDocumentTask<SrTopicUrlDTO, SrTopicUrlDTO> {
+
+  private static final Logger LOG = LogManager.getLogger(SrTopicArchivePageTask.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)